db/migrate/20230228215351_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] This will create `user_id` and a corresponding index, but as you can see in the ERD diagram, we should have a `author_id` column instead while letting Rails know that it is pointing to the Users table. Please see hint below 👇 
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```